File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function (sentence) {
    let arr = new Array(26).fill(false);

    for (let i = 0; i < sentence.length; i++) {
        let index = sentence[i].charCodeAt(0) - 'a'.charCodeAt(0);
        arr[index] = true;
    }

    for (let ele of arr) {
        if (!ele) return false;
    }

    return true;
};

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        boolean [] arr = new boolean[26];

        for(char ch : sentence.toCharArray()){
            int index = ch - 'a';
            arr[index] = true;
        }

        for(boolean boo : arr){
            if(!boo) return false;
        }
        return true;
    }
}